core/frontend/src/tile/RealityTileTree.ts

```diff
@@ -181,7 +185,7 @@ export class RealityTileTree extends TileTree {
 
   public prune(): void {
     const olderThan = BeTimePoint.now().minus(this.expirationTime);
-    this.rootTile.purgeContents(olderThan);
+    this.rootTile.purgeContents(olderThan, !ProcessDetector.isMobileBrowser);
```

Comment:
``ProcessDetector.isMobileBrowser`` is the simplest way I found to know when we are running on mobile, even though I am not sure it is very elegant.

Another way to do it would possibly be to add a boolean argument to this function and all its caller to set it from the TileAdmin, but I am not convinced about it either.

Reply:
This seems fine to me at least.